bower.json

```diff
@@ -10,9 +10,11 @@
   ],
   "main": [
     "main.scss",
-		"main.js"
+    "main.js"
```

Comment:
As the component doesn't have any JS functionality I think we should remove all the JS files such as the `main.js` and `teaser.js` and their corresponding tests, this would keep users from downloading code they do not require.